cilium-dbg/cmd/helpers.go

```diff
@@ -423,13 +424,8 @@ func getIpEnableStatuses() (bool, bool) {
 }
 
 func mergeMaps(m1, m2 map[string]interface{}) map[string]interface{} {
-	m3 := make(map[string]interface{})
```

Comment:
nit: could make this all `map[string]any`

Reply:
also ... nit: could be simplified down to just:
```go
return maps.Copy(maps.Clone(m1), m2)
```

Reply:
Thanks. I plan to address the `interface{}` to `any` conversion in a follow-up change across the entire code base. `modernize` does this as well but I kept it out to keep the amount of changes to an acceptable minimum for reviewers.

Reply:
> ```go
> return maps.Copy(maps.Clone(m1), m2)
> ```

This unfortunately won't work because `maps.Copy` doesn't return a value.